app/views/goods/index.html.erb

```diff
@@ -55,13 +54,19 @@ document.addEventListener("turbolinks:load",function(){
     index_input.easyAutocomplete(options);
 });
 </script>
-<div uk-grid>
-    <div class="uk-width-1-4@m">Reklám hely</div>
-    <div class="uk-width-1-2@m">
-        <%= form_tag filter_index_get_path, method: :get, id: "index_form", class: "uk-search uk-search-default uk-search-navbar" do %>
-            <input name="searched_phrase" id="index_input" class="uk-search-input" type="search" placeholder="Keresés...">
-            <button type="submit" form="index_form" class="uk-search-icon-flip" uk-search-icon></button>
+<div class = "uk-position-center uk-width-1-2@m">
+    <div id="main" class="uk-background-cover">
+        <%= form_tag filter_index_get_path, method: :get, id: "index_form", class: "uk-search uk-search-default uk-search-navbar uk-align-center uk-search-large uk-background-muted uk-border-rounded"  do %>
+            <input name="searched_phrase" id="index_input" class="uk-search-input uk-width-1" type="search" placeholder="Keresés...">
+            
+            <button id="index_search_button" type="submit" form="index_form" class="uk-search-icon-flip" uk-search-icon></button>
         <% end %>
     </div>
-    <div class="uk-width-1-4@m">Reklám hely</div>
+    <div align = "center" class = "uk-padding-large">
```

Comment:
Nincs align attribútum HTML-ben, ezt majd később ki lehet venni. Amúgy a középre igazításra van CSS attribútum, de valószínűlég a UI kit-tel is meg lehet oldani.